addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview.xml

```diff
@@ -38,14 +38,34 @@
                         </t>
                     </div>
                     <div class="o_ThreadNeedactionPreview_core">
-                        <span class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_inlineText text-truncate" t-att-class="{ 'o-empty': inlineLastNeedactionMessageAsOriginThreadBody.length === 0 }">
-                            <t t-if="threadNeedactionPreviewView.thread.lastNeedactionMessageAsOriginThread and threadNeedactionPreviewView.thread.lastNeedactionMessageAsOriginThread.author">
-                                <MessageAuthorPrefix
-                                    messageLocalId="threadNeedactionPreviewView.thread.lastNeedactionMessageAsOriginThread.localId"
-                                    threadLocalId="threadNeedactionPreviewView.thread.localId"
-                                />
-                            </t>
-                            <t t-esc="inlineLastNeedactionMessageAsOriginThreadBody"/>
+                        <span class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_inlineText text-truncate" t-att-class="{ 'o-empty': lastMessageNeedactionMessageContent.length === 0 }">
+                            <span class="o_ThreadNeedactionPreview_messageBody">
+                                <t t-if="lastMessageNeedactionMessageContent.trackingValue">
+                                    <span class="o_ThreadNeedactionPreview_trackingValue">
+                                        <span class="o_ThreadNeedactionPreview_trackingValueFieldName">
+                                            <t t-esc="lastMessageNeedactionMessageContent.trackingValue.changedFieldAsString"/>
+                                        </span>
+                                        <span class="o_ThreadNeedactionPreview_trackingValueOldValue">
+                                            <t t-esc="lastMessageNeedactionMessageContent.trackingValue.oldValueAsString"/>
+                                        </span>
+                                        <span class="o_ThreadNeedactionPreview_arrow">
+                                            <div class="o_ThreadNeedactionPreview_trackingValueSeparator fa fa-long-arrow-right mx-1" role="img"/>
+                                        </span>
+                                        <span class="o_ThreadNeedactionPreview_trackingValueNewValue">
+                                            <t t-esc="lastMessageNeedactionMessageContent.trackingValue.newValueAsString"/>
+                                        </span>
+                                    </span>
+                                </t>
```

Comment:
All this could be a component, to reduce complexity of `ThreadNeedactionPreviewComponent`.